### PR TITLE
add config file for customizing deck

### PR DESCRIPTION
**Pull request by ThunderDev1**
For now there's only a text property, but this will allow us to easily add other stuff in the futur (like an image/gif or sound file path).

### CODE REVIEW

File: app/src/bf-game/index.js [hunk 1]
```diff
@@ -5,8 +5,10 @@
  * state.
  */
 
-import { times, shuffle, deepClone } from "./utils";
-
+import { shuffle, deepClone } from "./utils";
+import { defaultDeck } from "./cards";
+import { CardType } from "./types";
+import { v4 as uuidv4 } from "uuid";
```
Comment (hissalht) [inline app/src/bf-game/index.js:11]: rhooo... not even making our own `uniqueId` function ?

File: app/src/bf-game/index.js [hunk 2]
```diff
@@ -283,7 +265,10 @@ export function insertPerish(gameState, perishNewPosition) {
   const playerHand = newGameState.hands[currentPlayerId];
 
   // Perish card is re-inserted into deck at an index chosen by the player
-  const perishCardIndex = playerHand.indexOf(CardType.Perish);
+  console.log("does this work ??");
```
Comment (hissalht) [inline app/src/bf-game/index.js]: to remove

**Review by hissalht — APPROVED** [review]

File: app/src/bf-game/index.js (hunk 2 above)
Comment (ThunderDev1) [reply app/src/bf-game/index.js]: Oops ^^'

File: app/src/bf-game/index.js (hunk 1 above)
Comment (ThunderDev1) [reply app/src/bf-game/index.js:11]: Nope, it's called lazyness ^^

**Pull request merged**